### PR TITLE
Describe how to use yapf, plus other updates to CONTRIBUTING.md.

### PR DESCRIPTION
**Pull request by jamessynge**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
+We prefer that all changes to POCS have an associated
+[GitHub Issue in the project](https://github.com/panoptes/POCS/issues)
+that explains why it is needed. This allows us to debate the best
+approach to address the issue before folks spend a lot of time
+writing code. If you are unsure about a possible contribution to
+the project, please contact the project owners about your idea;
+of course, an [issue](https://github.com/panoptes/POCS/issues) is a
+good way to do this.
 
+Please see the
```
Comment (wtgee) [inline CONTRIBUTING.md]: Let's move this line to the top. Maybe even above the "Getting Started"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -38,13 +49,21 @@ playground rules and follow them during all your contributions.
...
 # Code Formatting
 
 - All Python should use [PEP 8 Standards](https://www.python.org/dev/peps/pep-0008/)
    - Line length is set at 100 characters instead of 80.
    - It is recommended to have your editor auto-format code whenever you save a file 
       rather than attempt to go back and change an entire file all at once.
+   - You can also use
+     [yapf (Yet Another Python Formatter)](https://github.com/google/yapf)
+     for which POCS includes a style file (.style.yapf). For example:
+     ```bash
```
Comment (wtgee) [inline CONTRIBUTING.md]: Should we make a script for this?

Comment (jamessynge) [reply CONTRIBUTING.md]: Well, maybe. You can submit an issue if you like.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jamessynge) [reply CONTRIBUTING.md]: Done

**Comment by jamessynge** [comment]
Apparently needs your review again. Is there a github option to allow members of certain sub-teams to commit after an approval that requested a change?

**Comment by wtgee** [comment]
> Apparently needs your review again. Is there a github option to allow members of certain sub-teams to commit after an approval that requested a change?

It was set to automatically dismiss approvals if there were any new commits after the approval. I changed the setting so now an approval should stick.

File: CONTRIBUTING.md (hunk 2 above)
Comment (wtgee) [reply CONTRIBUTING.md]: We'll leave for now.

**Pull request merged**